content/guides/hosting/hosting-options/self-managed/ref-arch.md

```diff
@@ -56,23 +56,21 @@ Consider the following when you deploy a self-managed MySQL database:
 - **Monitoring.** The database should be monitored for load. If CPU usage is sustained at > 40% of the system for more than 5 minutes it is likely a good indication the server is resource starved.
 - **Availability.** Depending on your availability and durability requirements you might want to configure a hot standby on a separate machine that streams all updates in realtime from the primary server and can be used to failover to in the event that the primary server crashes or become corrupted.
 
-
 ### Object storage
-W&B requires object storage with Pre-signed URL and CORS support, deployed in Amazon S3, Azure Cloud Storage, Google Cloud Storage, or a storage service compatible with Amazon S3.service) 
+W&B requires object storage with Pre-signed URL and CORS support, deployed in Amazon S3, Azure Cloud Storage, Google Cloud Storage, or a storage service compatible with Amazon S3) 
 
 ### Versions
 * Kubernetes: at least version 1.29.
-* MySQL: at least 8.0.
-
+* MySQL: at least 8.0.0. Choose a MySQL ["General availability"](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/) release.
 
 ### Networking
 
-In a deployment connected a public or private network, egress to the following endpoints is required during installation _and_ during runtime:
-    * `https://deploy.wandb.ai`
-    * `https://charts.wandb.ai`
-    * `https://docker.io`
-    * `https://quay.io`
-    * `https://gcr.io`
+Egress to the following endpoints is required during installation _and_ during runtime:
+* `https://deploy.wandb.ai`
```

Comment:
Please update these to be links, using the same syntax as in https://github.com/wandb/docs/pull/1020/files.